requirements.txt

```diff
@@ -1,5 +1,5 @@
--e git://github.com/caciviclab/django-calaccess-raw-data.git@0dd6b6913e4a00b49d1b9c9bf9829ed83f4587a3#egg=django-calaccess-raw-data
+-e git://github.com/caciviclab/django-calaccess-raw-data.git#egg=django-calaccess-raw-data
```

Comment:
As I mentioned in #71 , it is important that we keep the `@0dd6b6913e4a00b49d1b9c9bf9829ed83f4587a3` version pin here.